### PR TITLE
Subsonic Music Playback

### PR DESCRIPTION
**Pull request by transcental**
Add support for playing music on the host device from a Subsonic server through Rhea.

### CODE REVIEW

File: server/blueprints/music.py [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+    except Exception:  # noqa
+        # We use a broad exception as there are a variety of Connection* errors that can be raised
+        print_warn("Failed to connect to Subsonic server")
+        return abort(500, "Failed to connect to Subsonic server")
+
+
+@music.route("/")
+def index():
+    print_info("Attempting to connect to Subsonic server")
+    if subsonic.ping():
```
Comment (Coding-Cactus) [inline server/blueprints/music.py]: Unnecessary due to before_request()

File: server/blueprints/music.py [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+                "artist": song.artist,
+                "album": song.album,
+                "track": song.track,
+                "cover": song.cover,
+                "duration": song.duration,
+                "genre": song.genre,
+                "year": song.year,
+                "id": song.id,
+            }
+            for song in music_queue
```
Comment (Coding-Cactus) [inline server/blueprints/music.py]: Could potentially be very large

File: server/blueprints/music.py [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+            and (current_time / length) > 0.5
+            and not scrobbled
+        ):
+            subsonic.scrobble(current_song.id, True)
+            scrobbled = True
+        else:
+            # Dynamically adjust sleep interval based on system load or other conditions.
+            # We are sleeping because we do not want to constantly check the play status,
+            # but we also do not want to wait too long to play the next song.
+            sleep_interval = random.uniform(0.5, 1.5)
```
Comment (Coding-Cactus) [inline server/blueprints/music.py]: > based on system load or other conditions

Not too sure you're achieving that

**Comment by DillonB07** [comment]
(I did not mean to request a review, but if you'd like to, then I'd appreciate it)

**Pull request merged**